mzLib/mzLib.nuspec

```diff
@@ -37,48 +37,48 @@
-    <file src="Readers\bin\x64\Release\net6.0\Thermo\ThermoFisher.CommonCore.Data.dll" target="lib\net6.0-windows7.0" />
-    <file src="Readers\bin\x64\Release\net6.0\Thermo\ThermoFisher.CommonCore.MassPrecisionEstimator.dll" target="lib\net6.0-windows7.0" />
-    <file src="Readers\bin\x64\Release\net6.0\Thermo\ThermoFisher.CommonCore.RawFileReader.dll" target="lib\net6.0-windows7.0" />
-    <file src="UsefulProteomicsDatabases\bin\x64\Release\net6.0\UsefulProteomicsDatabases.dll" target="lib\net6.0-windows7.0" />
-    <file src="Transcriptomics\bin\x64\Release\net6.0\Transcriptomics.dll" target="lib\net6.0-windows7.0" />
-    <file src="Transcriptomics\bin\x64\Release\net6.0\Digestion\rnases.tsv" target="lib\net6.0-windows7.0" />
+    <!--net8.0 target-->
+    <file src="BayesianEstimation\bin\x64\Release\net8.0\BayesianEstimation.dll" target="lib\net8.0" />
+    <file src="Chemistry\bin\x64\Release\net8.0\Chemistry.dll" target="lib\net8.0" />
+    <file src="FlashLFQ\bin\x64\Release\net8.0\FlashLFQ.dll" target="lib\net8.0" />
+    <file src="MassSpectrometry\bin\x64\Release\net8.0\MassSpectrometry.dll" target="lib\net8.0" />
+    <file src="MzIdentML\bin\x64\Release\net8.0\MzIdentML.dll" target="lib\net8.0" />
+    <file src="SpectralAveraging\bin\x64\Release\net8.0\SpectralAveraging.dll" target="lib\net8.0" />
+    <file src="MzLibUtil\bin\x64\Release\net8.0\MzLibUtil.dll" target="lib\net8.0" />
+    <file src="Omics\bin\x64\Release\net8.0\Omics.dll" target="lib\net8.0" />
+    <file src="PepXML\bin\x64\Release\net8.0\PepXML.dll" target="lib\net8.0" />
+    <file src="Proteomics\bin\x64\Release\net8.0\Proteomics.dll" target="lib\net8.0" />
+    <file src="Proteomics\bin\x64\Release\net8.0\ProteolyticDigestion\proteases.tsv" target="lib\net8.0\ProteolyticDigestion" />
+    <file src="UsefulProteomicsDatabases\bin\x64\Release\net8.0\UniProtKB_columnNamesForProgrammaticAccess.txt" target="lib\net8.0" />
+    <file src="Readers\bin\x64\Release\net8.0\Readers.dll" target="lib\net8.0" />
+    <file src="Readers\bin\x64\Release\net8.0\Thermo\ThermoFisher.CommonCore.BackgroundSubtraction.dll" target="lib\net8.0" />
+    <file src="Readers\bin\x64\Release\net8.0\Thermo\ThermoFisher.CommonCore.Data.dll" target="lib\net8.0" />
+    <file src="Readers\bin\x64\Release\net8.0\Thermo\ThermoFisher.CommonCore.MassPrecisionEstimator.dll" target="lib\net8.0" />
+    <file src="Readers\bin\x64\Release\net8.0\Thermo\ThermoFisher.CommonCore.RawFileReader.dll" target="lib\net8.0" />
+    <file src="UsefulProteomicsDatabases\bin\x64\Release\net8.0\UsefulProteomicsDatabases.dll" target="lib\net8.0" />
+    <file src="Transcriptomics\bin\x64\Release\net8.0\Transcriptomics.dll" target="lib\net8.0" />
+    <file src="Transcriptomics\bin\x64\Release\net8.0\Digestion\rnases.tsv" target="lib\net8.0" />
+    <!--net8.0-windows target with mzPlot-->
+    <file src="mzPlot\bin\x64\Release\net8.0-windows\mzPlot.dll" target="lib\net8.0-windows8.0" />
```

Comment:
`net8.0-windows`

Reply:
end of these lines should be changed to windows7.0